app/assets/stylesheets/application.scss

```diff
@@ -7,3 +7,63 @@
 @import "base/base";
 
 @import "layout";
+
+/* 
+  Variables
+*/
+
+$black: #000;
+$grey: #ddd;
+
+$alert: #de1b31;
+$error: #de1b31;
+$notice: #000;
+$success: #2adc55;
+
+.nav {
+
+  .menu {
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets